biospecimen-tracking/api/pom.xml

```diff
@@ -0,0 +1,91 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.phenotips</groupId>
+        <artifactId>patient-data-biospecimen-tracking</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>patient-data-biospecimen-tracking-api</artifactId>
+    <name>PhenoTips for NIAID - Biospecimen Tracking API</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>application-configuration-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>patient-data-api</artifactId>
+            <version>${phenotips.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.xwiki.commons</groupId>
+            <artifactId>xwiki-commons-component-api</artifactId>
+            <version>${xwiki.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.xwiki.platform</groupId>
+            <artifactId>xwiki-platform-model</artifactId>
+            <version>${xwiki.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.xwiki.platform</groupId>
+            <artifactId>xwiki-platform-bridge</artifactId>
+            <version>${xwiki.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.xwiki.commons</groupId>
+            <artifactId>xwiki-commons-tool-test-component</artifactId>
+            <version>${xwiki.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.xwiki.platform</groupId>
+            <artifactId>xwiki-platform-rest-server</artifactId>
+            <version>${xwiki.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
+            <version>4.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>1.10.19</version>
```

Comment:
Mockito should be a transitive dependency through the XW test projects. See here for an example: https://github.com/phenotips/phenotips/blob/a6e23ddfd9dc553de01b345f582fe8381c38a40b/components/solr-access-service/api/pom.xml#L81-L93

Also, test dependencies should specify `<scope>test</scope>`, otherwise they'll be included uneccessarily in deployments.

More information is available here:
http://dev.xwiki.org/xwiki/bin/view/Community/Testing